common/production_method_groups/babylon_urban.txt

```diff
@@ -1,7 +1,22 @@
-﻿pmg_automobile_roads = {
+﻿pmg_road_transport = {
 	texture = "gfx/interface/icons/generic_icons/mixed_icon_refining.dds"
 	production_methods = {
-		pm_no_automobile_roads
-		pm_public_roads
+		pm_railway_prioritization
+		pm_hand_carts
+		pm_wagons
+		pm_steam_trucks
+		pm_diesel_trucks
+	}
+}
+
+pmg_public_transport = {
+	texture = "gfx/interface/icons/generic_icons/mixed_icon_refining.dds"
+	production_methods = {
+		pm_no_public_transport
+		pm_rickshaws
+		pm_stagecoaches
+		pm_trams
+		pm_steam_buses
+		pm_motor_buses
 	}
```

Comment:
You're overwriting this from vanilla, which means that I want it in the vanilla file (for future patches aka maintainance)

Reply:
I've put this pmg into the vanilla files.